### PR TITLE
Hide un-useful fields in shelter index view. Added two new private fields.

### PR DESCRIPTION
**Pull request by rossettistone**
Per https://github.com/Irma-Response/irma-api/issues/5, this PR:

Removes columns from the shelter index view that are not useful. These columns are still available in the "show" and "update" views for shelters.
Adds two new columns, private_sms and private_volunteer_data_mgr

### CODE REVIEW

File: app/controllers/shelters_controller.rb
Comment (tdooner) [inline app/controllers/shelters_controller.rb:2]: Just for consistency, could you use a `before_action` for the index headers too?

File: app/controllers/shelters_controller.rb
Comment (tdooner) [inline app/controllers/shelters_controller.rb:87]: This could totally be done separately, but here's a thought:

```ruby
def set_headers
  @columns = Shelter::ColumnNames
  @headers = Shelter::HeaderNames

  if admin?
    # add admin columns
  end

  if params[:action] == 'index'
    # remove index columns
  end

  @headers.map!(&:titleize)
end
```

File: app/controllers/shelters_controller.rb
Comment (tdooner) [inline app/controllers/shelters_controller.rb:86]: Is this a typo? Should it be HeaderNames?

File: app/controllers/shelters_controller.rb
Comment (tdooner) [inline app/controllers/shelters_controller.rb:81]: Why the local variable?

**Review by tdooner — APPROVED** [review]
Looks generally good. I'm going to merge this and deploy it, but if you still have more time to address the feedback, that would be great.

File: app/controllers/shelters_controller.rb
Comment (rossettistone) [reply app/controllers/shelters_controller.rb:2]: Yep, will do.

**Comment by tdooner** [comment]
Actually, the ColumnNames/HeaderNames thing seems like maybe a bug, so I'm going to hold off merging until I hear back on that.

File: app/controllers/shelters_controller.rb
Comment (rossettistone) [reply app/controllers/shelters_controller.rb:86]: In this particular method, I'm not a huge fan of `HeaderNames`, as it's not explicit that it refers *only* to the headers of the set of default columns in `ColumnNames`. Given that we have references to three different sets of column names (`ColumnNames`, `PrivateFields`, and now `IndexHiddenColumnNames`), it seemed helpful to keep everything explicit here and just call `:titleize` on the raw column names right here.

File: app/controllers/shelters_controller.rb
Comment (rossettistone) [reply app/controllers/shelters_controller.rb:81]: It's ever-so-slightly DRY-er, and makes for parity with `set_index_headers` below.

**Review by tdooner — APPROVED** [review]

**Comment by tdooner** [comment]
Awesome, thanks!

**Pull request merged**
